browser-test/bin/wait_for_server_start_and_run_tests.sh

```diff
@@ -4,15 +4,37 @@ set -euo pipefail
 
 START_TIME=$(date +%s)
 DEADLINE=$(($START_TIME + 500))
-SERVER_URL="http://civiform:9000"
+
+# Allow callers to override BASE_URL if they want (e.g. for run-browser-tests-local).
+# Defaults civiform:9000 when running from within docker.
+export BASE_URL="${BASE_URL:-http://civiform:9000}"
+
+export TEST_USER_AUTH_STRATEGY=fake-oidc
+export TEST_USER_LOGIN=testuser
+export TEST_USER_PASSWORD=anotsecretpassword
+# The display name returned by test_oidc_provider.js is <username>@example.com.
+export TEST_USER_DISPLAY_NAME=testuser@example.com
+
+if ! output="$(node -v)"; then
+  echo output
+  echo "You must have node installed locally to run this command. Go to https://nodejs.org/en/download/package-manager/ for installation instructions."
+  exit 1
+fi
+
+# Sanity check that this script was called from within browser-test directory.
+if [ ! -d "image_snapshots" ]; then
```

Comment:
is image_snapshots correct? the comment and error messages say browser-test

Reply:
Clarified comment. I'm checking that image_snapshots directory exists as a proxy for checking from which directory the script is called. I can't check name of the current directory as in some docker runs we map browser-test to a differently named directory.